documentation/src/main/asciidoc/topics/ref_rest_counters.adoc

```diff
@@ -41,32 +41,6 @@ GET /rest/v2/counters/{counterName}/config
 
 {brandname} responds with the counter configuration in JSON format.
 
-
-[id='rest_server_counters_add']
```

Comment:
@oraNod please pay attention, that I've removed this call as this service not implemented as expected